internal/cmd/argus/grafana/public-read-access/public_read_access.go

```diff
@@ -0,0 +1,33 @@
+package publicreadaccess
+
+import (
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/cmd/argus/grafana/public-read-access/disable"
+	"github.com/stackitcloud/stackit-cli/internal/cmd/argus/grafana/public-read-access/enable"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+
+	"github.com/spf13/cobra"
+)
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "public-read-access",
+		Short: "Enable or disable public read access for Grafana in Argus instances",
+		Long: fmt.Sprintf("%s\n%s",
+			"Enable or disable public read access for Grafana in Argus instances.",
+			"When enabled, anyone can access the Grafana dashboards without of the instance logging in. Otherwise, a login is required.",
```

Comment:
```suggestion
			"When enabled, anyone can access the Grafana dashboards without logging in. Otherwise, a login is required.",
```

Reply:
Oh, I meant to say `... Grafana dashboards of the instance without ...`